lib/ansible/modules/system/aix_lvol.py

```diff
@@ -0,0 +1,364 @@
+    lv: testlv
+    state: absent
+'''
+
+RETURN = '''
+msg:
+  type: string
+  description: A friendly message describing the task result.
+  returned: always
+  sample: Logical volume testlv created.
+'''
+
+import re
+
+def convert_size(size):
+    base_size = None
+    size_factor = None
+
+    match = re.search("(\d+)", size)
+    if match is not None:
+        base_size = int(match.group(1))
+
+    match = re.search("(\w)", size)
+    if match is not None:
+
+        if match.group(1) == "G":
+            size_factor=1024
+        if match.group(1) == "T":
+            size_factor=1024*1024
+
+    if base_size is None:
+        return None
+
+    if size_factor is None:
+        return base_size
+
+    return base_size * base_factor
+
+def round_ppsize(x, base=16):
+    new_size = int(base * round(float(x)/base))
+    if new_size < x:
+        new_size += base
+    return new_size
+
+
+def parse_lv(data):
+    name=None
+
+    for line in data.splitlines():
+        match = re.search("LOGICAL VOLUME:\s+(\w+)\s+VOLUME GROUP:\s+(\w+)", line)
+        if match is not None:
+            name = match.group(1)
+            vg = match.group(2)
+            continue
+        match = re.search("LPs:\s+(\d+).*PPs", line)
+        if match is not None:
+            lps = int(match.group(1))
+            continue
+        match = re.search("PP SIZE:\s+(\d+)", line)
+        if match is not None:
+            pp_size = int(match.group(1))
+            continue
+        match = re.search("INTER-POLICY:\s+(\w+)", line)
+        if match is not None:
+            policy = match.group(1)
+            continue
+
+    if not name:
+        return None
+
+    size = lps * pp_size
+
+    return {'name': name, 'vg': vg, 'size': size, 'policy': policy}
+
+def parse_vg(data):
+
+    for line in data.splitlines():
+
+        match = re.search("VOLUME GROUP:\s+(\w+)", line)
+        if match is not None:
+            name = match.group(1)
+            continue
+
+        match = re.search("TOTAL PP.*\((\d+)", line)
+        if match is not None:
+            size = int(match.group(1))
+            continue
+
+        match = re.search("PP SIZE:\s+(\d+)", line)
+        if match is not None:
+            pp_size = int(match.group(1))
+            continue
+
+        match = re.search("FREE PP.*\((\d+)", line)
+        if match is not None:
+            free = int(match.group(1))
+            continue
+
+    return {'name': name, 'size': size, 'free': free, 'pp_size': pp_size}
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            vg=dict(required=True),
+            lv=dict(required=True),
+            lv_type=dict(required=False),
+            size=dict(type='str'),
+            opts=dict(type='str'),
+            copies=dict(type='str'),
+            state=dict(choices=["absent", "present"], default='present'),
+            policy=dict(choices=["maximum", "minimum"], default='maximum'),
+            pvs=dict(type='str')
```

Comment:
You can use `type='list'` with `default=list()`.
Later, you can use it like `' '.join(pvs)`.

https://github.com/natefoo/ansible/blob/599eb6115e53fd6fd15770a4c7ef292177d5b888/lib/ansible/module_utils/basic.py#L1525